### PR TITLE
Form to Migrate data between Image stores

### PR DESCRIPTION
**Pull request by Pearl1594**
Enable migration of data between secondary storage pools - addresses feature: https://github.com/apache/cloudstack/pull/4053

### CODE REVIEW

**Comment by vladimirpetrov** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@Pearl1594 please ping me when this is ready for review

**Comment by Pearl1594** [comment]
@rhtyd It's ready for review

**Comment by rhtyd** [comment]
@Pearl1594 does it cover for all cases in https://github.com/apache/cloudstack/pull/4053 ?

**Comment by Pearl1594** [comment]
Yes @rhtyd

**Comment by rhtyd** [comment]
As soon we have some test confirmation on this we can merge this PR cc @vladimirpetrov @borisstoyanov

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by vladimirpetrov** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@Pearl1594 can you fix the conflict please? Is it ready for code review?

**Comment by Pearl1594** [comment]
@blueorangutan package

**Comment by Pearl1594** [comment]
@rhtyd resolved the conflict. It's ready for review

File: src/config/section/infra/secondaryStorages.js [hunk 1]
```diff
@@ -21,8 +21,8 @@ export default {
   icon: 'picture',
   docHelp: 'adminguide/storage.html#secondary-storage',
   permission: ['listImageStores'],
-  columns: ['name', 'url', 'protocol', 'scope', 'zonename'],
-  details: ['name', 'id', 'url', 'protocol', 'provider', 'scope', 'zonename'],
+  columns: ['name', 'url', 'protocol', 'scope', 'zonename', 'readonly'],
+  details: ['name', 'id', 'url', 'protocol', 'provider', 'scope', 'zonename', 'readonly'],
```
Comment (davidjumani) [inline src/config/section/infra/secondaryStorages.js:25]: Check for readonly in api params

**Comment by davidjumani** [comment]
LGTM. Needs to be tested on the specific setup

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@Pearl1594 can you fix the merge conflict? Thanks.

**Comment by Pearl1594** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
Please merge latest master and fix any conflicts? Thanks.

**Comment by Pearl1594** [comment]
@blueorangutan package

**Comment by Pearl1594** [comment]
@blueorangutan package

**Comment by Pearl1594** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@Pearl1594 can you fix the conflicts? @borisstoyanov @vladimirpetrov are you lgtm on this?

**Comment by Pearl1594** [comment]
@blueorangutan package

**Comment by Pearl1594** [comment]
@blueorangutan package

**Comment by vladimirpetrov** [comment]
I tested this as part of FR76 Secondary storage management and it is LGTM.

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by Pearl1594** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@davidjumani can you review and advise if we can merge this?

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by Pearl1594** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@Pearl1594 @davidjumani can you advise when this is ready for merging? Thnx

**Comment by Pearl1594** [comment]
@blueorangutan package

**Comment by Pearl1594** [comment]
@davidjumani ready for review

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
Has this been tested @Pearl1594 @vladimirpetrov ?

**Comment by davidjumani** [comment]
Tested on the appropriate env. Works fine!

**Comment by Pearl1594** [comment]
yes @rhtyd,

**Pull request merged**